Source/AssetRipper.Processing/AnimationClips/AnimationCache.cs

```diff
@@ -8,9 +8,9 @@ namespace AssetRipper.Processing.AnimationClips;
 
 public sealed class AnimationCache
 {
-	public IReadOnlyList<IAvatar> CachedAvatars { get; }
-	public IReadOnlyList<IAnimator> CachedAnimators { get; }
-	public IReadOnlyList<IAnimation> CachedAnimations { get; }
+	public IAvatar[] CachedAvatars { get; }
+	public IAnimator[] CachedAnimators { get; }
+	public IAnimation[] CachedAnimations { get; }
```

Comment:
I originally used `IReadOnlyList<T>` for encapsulation and immutability. Please make these into `private readonly` fields and `ReadOnlySpan<T>` properties.